lib/views/pages/intro/constantly_moving_lottie.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:lottie/lottie.dart';
+
+class ConstantlyMoving extends StatefulWidget {
+  const ConstantlyMoving({super.key, required this.asset});
+
+  final String asset;
+
+  @override
+  State<ConstantlyMoving> createState() => _ConstantlyMovingState();
+}
+
+class _ConstantlyMovingState extends State<ConstantlyMoving> {
+  late final Future<LottieComposition> _composition;
+
+  @override
+  void initState() {
+    super.initState();
+    _composition = AssetLottie(widget.asset).load();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        FutureBuilder<LottieComposition>(
+          future: _composition,
+          builder: (context, snapshot) {
+            var composition = snapshot.data;
+            if (composition != null) {
+              return SizedBox(
+                  width: 380,
+                  height: 380,
+                  child: Lottie(composition: composition));
+            } else {
+              return const Center(child: CircularProgressIndicator());
```

Comment:
これだと、読み込まれるまで小さいローディングが出るので、読み込み完了時に画面にガタつきが発生します。
SizedBox は feaatureBuilder の外に出してあげましょう。あとColumnは要らなそう。

```suggestion
return SizedBox(
  width: size,
  height: size,
  child: FutureBUilder<LottieComposition>(
    ... ( SizedBox は削除 )
  ),
);
```